media-plugins/distrho-ports/distrho-ports-99999999.ebuild

```diff
@@ -22,41 +22,26 @@ REQUIRED_USE="|| ( lv2 vst )"
 
 RDEPEND="media-libs/alsa-lib
 	media-libs/freetype
-	media-libs/ladspa-sdk
 	virtual/opengl
 	x11-libs/libX11
 	x11-libs/libXext
 	x11-libs/libXcursor
 	x11-libs/libXrender"
-DEPEND="${RDEPEND}
-	dev-util/premake:3"
-
-QA_PRESTRIPPED="
-	/usr/lib/lv2/.*
-	/usr/lib/vst/.*
-"
+DEPEND="${RDEPEND}"
 
 src_prepare() {
-	default
-	scripts/premake-update.sh linux
-}
+	# Remove stripping of binaries
+	sed -i -e "/'-Wl,--strip-all',/d" meson.build || die "sed failed"
 
-src_compile() {
-	if use lv2; then
-		emake lv2
-	fi
-	if use vst; then
-		emake vst
-	fi
+	default
 }
 
-src_install() {
-	emake DESTDIR="${D}" PREFIX="/usr" install
-	if use !lv2; then
-		rm -rf "${D}"/usr/lib/lv2
-	fi
-	if use !vst; then
-		rm -rf "${D}"/usr/lib/vst
-	fi
-	rm -rf "${D}"/usr/src
+src_configure() {
+	local emesonargs=(
+		-Doptimizations=false
```

Comment:
This disables default for enabling SSE2/3 etc, see https://github.com/DISTRHO/DISTRHO-Ports/blob/master/meson.build#L129 I think this is what we should be doing, right? To allow the user to choose.

Reply:
`CFLAGS` are honored, it's using the `-march=core-avx-i -O2` from my `make.conf` :)